scripts/tests/assert_cpu_jitter_entropy.rs

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env -S cargo +nightly -Zscript
```

Comment:
Just wondering: why is this a standalone script and not just a normal unit test?

Reply:
I wanted to simulate a consumer who depends on aws-lc-rs setting the `AWS_LC_FIPS_SYS_CPU_JITTER_ENTROPY` environment variable then expecting jitter entropy to be available. This script provides that setup in a compact form. (We also have couple of unit tests for coverage.)